samples/stock/plotoptions/series-datagrouping-simple-approximation/demo.html

```diff
@@ -0,0 +1,5 @@
+<div id="container" style="height: 400px; min-width: 600px"></div>
+
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script type="text/javascript" src="https://www.highcharts.com/samples/data/three-series-1000-points.js"></script>
```

Comment:
Is it necessary? `type="text/javascript"`

Reply:
I think it's not necessary, I copied that from [approximation demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/plotoptions/series-datagrouping-approximation)

Reply:
Not necessary.